book/src/integrations/oauth2.md

```diff
@@ -286,6 +286,29 @@ kanidm system oauth2 disable-localhost-redirects <name>
 kanidm system oauth2 enable-localhost-redirects mywebapp
 ```
 
+## Alternate Redirect Origins
+
+Some services may have a website url as well as native applications. These native applications
+require alternate redirection urls to be configured so that after an oauth2 exchange, the system can
+redirect to the native application.
```

Comment:
```suggestion
Some services may have a website URL as well as native applications. These native applications
require alternate redirection URLs to be configured so that after an OAuth2 exchange, the system can
redirect to the native application.
```